.gitignore

```diff
@@ -108,3 +108,4 @@ data/
 
 # PostgreSQL files
 perfil/mydb
+perfil/db
```

Comment:
Do we need both, `perfil/db` and `perfil/mydb`? We could adopt one of them and consider that one as the project default.